src/site/layout/css.rs

```diff
@@ -43,84 +38,27 @@ pub fn get_css_link(path_prefix: &Option<String>, release_tag: &str) -> Result<S
     // by falling back to fetching said version from GitHub.
     if cfg!(debug_assertions) && release_tag == ORANDA_CSS_TAG {
         // If we're running in a local environment, we fetch a Tailwind binary and compile the CSS
         // on the spot. This is useful if we're working on oranda-css locally.
-        build_css(dist_dir)
-    } else {
-        // If we're running in a released binary, we fetch the latest tag (or whichever tag is
-        // specified in the configuration).
-        fetch_css(dist_dir, release_tag)
-    }
-}
-
-pub fn build_css(dist_dir: &str) -> Result<()> {
-    // Fetch our cache dir
-    let project_dir = ProjectDirs::from("dev", "axo", "oranda")
-        .expect("Unable to create cache dir for downloading Tailwind!");
-    let cache_dir = project_dir.cache_dir();
-    // Figure out our target "double" (tailwind has weird naming around this)
-    let double = match (env::consts::OS, env::consts::ARCH) {
-        ("linux", "x86_64") => "linux-x64",
-        ("linux", "aarch64") => "linux-arm64",
-        ("linux", "arm") => "linux-armv7",
-        ("macos", "x86_64") => "macos-x64",
-        ("macos", "aarch64") => "macos-arm64",
-        ("windows", "x86_64") => "windows-x64.exe",
-        ("windows", "aarch64") => "windows-arm64.exe",
-        _ => "linux-x64",
-    };
-    let mut binary_path = Utf8PathBuf::from(cache_dir.display().to_string());
-    LocalAsset::create_dir_all(&binary_path)?;
-    binary_path.push(format!("tailwindcss-{double}"));
-    if !binary_path.exists() {
-        // Fetch the binary from GitHub if it doesn't exist
-        tracing::info!("Fetching Tailwind binary from GitHub release...");
-        let url = format!(
-            "https://github.com/tailwindlabs/tailwindcss/releases/latest/download/tailwindcss-{double}"
-        );
-        let handle = tokio::runtime::Handle::current();
-        let response = handle.block_on(reqwest::get(url))?;
-        let bytes = handle.block_on(response.bytes())?;
-        let file = LocalAsset::new(&binary_path, Vec::from(bytes))?;
-        file.write(
-            binary_path
-                .parent()
-                .expect("Tailwind binary path has no parent!?"),
-        )?;
-
-        // On non-Windows platforms, we need to mark the file as executable
-        #[cfg(target_family = "unix")]
+        Ok(oranda_generate_css::build_css(dist_dir)?)
+    } else if maybe_env.is_some() && release_tag == ORANDA_CSS_TAG {
+        // If we're in a release build, we can inline a file generated by oranda's build script.
+        // Don't even build this if we're not in a release build so that Clippy et al don't complain
+        #[cfg(not(debug_assertions))]
         {
-            use std::os::unix::prelude::PermissionsExt;
-            let user_execute = std::fs::Permissions::from_mode(0o755);
-            std::fs::set_permissions(&binary_path, user_execute)?;
+            let css = include_str!("../../../oranda-css/dist/oranda.css");
```

Comment:
Is this not a compiler error for non-release builds? I would have thought all include_strs are resolved statically, so the fact that it's in a branch doesn't matter...

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=8441ab4ff8c4f663a7c39d8ed947da3f

Oh I see you wrapped this in `#[cfg(not(debug_assertions))]`... hmmm... I guess that should work? But at that point you might as well write the code as:

```rust
if release_tag == ORANDA_CSS_TAG {
    #[cfg(debug_assertions)]
    {
        // build css
    }
    #[cfg(not(debug_assertions)]
    {
        // write static css
        include_str!("../../oranda-css/dist/oranda.css")
    }
} else {
    // fetch css
}
```

And just not have the env var at all.

Reply:
> But at that point you might as well write the code as: <snip>

this makes sense to my sleep-addled brain, i'm pretty sure there was some objection i had to this path a couple hours earlier, i'll see if i remember tomorrow